tests/test_wave_02.py

```diff
@@ -27,7 +26,32 @@ def test_empty_watched_average_rating_is_zero():
     # Assert
     assert average == pytest.approx(0.0)
 
-@pytest.mark.skip()
+def test_calculates_watched_average_rating():
```

Comment:
Nice, I love seeing tests added when the existing tests don't cover the behavior. 😄